recipe/meta.yaml

```
@@ -23,9 +23,9 @@ requirements:
     - python >=3.6
     - absl-py >=0.9.0
     - dm-tree >=0.1.5
-    - jax >=0.1.55
+    - jax >=0.4.6
     - jaxlib >=0.1.37
-    - numpy >=1.18.0
+    - numpy >=1.25.0
```

you sure about that one? it's super recent. Maybe it risk colliding with other package dependencies.
I think recent jax versions only require numpy `1.22` or higher, so you could set this?

```suggestion
    - numpy >=1.22.0
```

---

Just FYI, I am not the right person to know about numpy versions, but ]I took that number from https://github.com/deepmind/chex/blob/master/requirements/requirements.txt.